src/App.css

```diff
@@ -218,16 +219,74 @@ div {
 }
 
 .BooksButton {
+  padding: 0.9rem 1.75rem;
   color: black;
-  background-color: white;
-  outline: black;
+  background-color: green;
+  outline: none;
+  border: none;
+  border-radius: 5rem;
+  font-size: 1.1rem;
+  cursor: pointer;
+  font-weight: 600;
+  transition: 0.2s;
+}
+
+.DeleteButton {
+  margin-left: auto;
+  margin-right: 0px;
+  margin-top: 10px;
+  padding: 0.9rem 1.75rem;
+  color: white;
+  background-color: red;
+  outline: none;
+  border: none;
+  border-radius: 5rem;
+  font-size: 1.1rem;
+  cursor: pointer;
+  font-weight: 600;
+  transition: 0.2s;
+
+}
+
+.BasicButton {
   padding: 0.9rem 1.75rem;
+  color: white;
+  background-color: #1565c0;
+  outline: none;
+  border: none;
   border-radius: 5rem;
   font-size: 1.1rem;
   cursor: pointer;
   font-weight: 600;
   transition: 0.2s;
 }
 
+table {
+  border-collapse: collapse;
+  width: 100%;
+}
 
+th {
+  background-color: #f2f2f2;
+  border: 1px solid #dddddd;
+  padding: 8px;
+  text-align: left;
+}
 
+td {
+  border: 1px solid #dddddd;
+  padding: 8px;
+}
+
+tr:nth-child(odd) {
+  background-color: #f9f9f9;
+}
+
+tbody {
+  border: 1px solid #dddddd;
+}
+
+table {
+  border-radius: 8px;
+  overflow: hidden;
+}
```

Comment:
The both table css lines and the td line are what mess up the home page.

Reply:
I updated to put everything in CSS class. Looks to be working correctly now.